src/package.json

```
@@ -13,7 +13,7 @@
   "author": "Marek Kulik <marek.kulik@hotmail.com>",
   "license": "MIT",
   "dependencies": {
-    "discord.js": "^11.2.0",
+    "discord.js": "^12.5.0",
```

12.5.1 was released yesterday
```suggestion
    "discord.js": "^12.5.1",
```